content/swiftui/concepts/navigation/navigation.md

```diff
@@ -0,0 +1,152 @@
+---
+Title: 'Navigation'
+Description: 'It is the ability to navigate to diffrent views from the root view in the app.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+  - 'iOS'
+CatalogContent:
+  - 'learn-iOS-course'
+  - 'path/iOS-path'
+---
+
+**Navigation** is the ability to navigate to diffrent views from the root view in the app. In SwiftUI, `NavigationStack` is used to enable the root view to present views over it, but in iOS 15 and less `NavigationView` was used in the past but it's deprecated since iOS 16 and introduced `NavigationStack`. 
```

Comment:
```suggestion
**Navigation** is the ability to navigate to different views from the root view in the app. In SwiftUI, `NavigationStack` is used to enable the root view to present views over it.  In iOS 15 and before, `NavigationView` was used, but it's been deprecated since iOS 16 introduced `NavigationStack`. 
```